core/wiki/config/OfficialPluginLibrary.tid

```diff
@@ -1,6 +1,6 @@
-title: $:/config/OfficialPluginLibrary
-tags: $:/tags/PluginLibrary
-url: https://tiddlywiki.com/library/v5.3.3/index.html
 caption: {{$:/language/OfficialPluginLibrary}}
+tags: $:/tags/PluginLibrary
+title: $:/config/OfficialPluginLibrary
+url: https://tiddlywiki5-git-plugin-stability-badges-jermolenes-projects.vercel.app/library/v5.3.3/index.html
 
-{{$:/language/OfficialPluginLibrary/Hint}}
+Plugin library for https://tiddlywiki5-git-plugin-stability-badges-jermolenes-projects.vercel.app
```

Comment:
@Jermolene -- I think there is a problem with this URL link.

Reply:
Oops thanks @pmario that was for the preview, I'll remove it